scripts/audit-consumers.ts

```diff
@@ -0,0 +1,201 @@
+import zlib from "node:zlib";
+import stream from "node:stream/promises";
+
+import * as cheerio from "cheerio";
+import { Octokit } from "@octokit/rest";
+
+import { features as webFeatures } from "../index.js";
+
+// Yield all entries from chromestatus.com. Because the newest entry is returned
+// first and entries might be created while we're iterating, it's possible that
+// the same entry is yielded multiple times. If this is a problem they have to be
+// deduplicated by ID.
+async function* chromeStatusFeatures() {
+  const num = 500;
+  for (let start = 0 /* nothing here */; ; start += num) {
```

Comment:
```suggestion
  for (let start = 0; /* nothing here */ ; start += num) {
```

Reply:
Turns out that Prettier required this style. I'll remove the comment, the result is OK enough to not try to work around.